### PR TITLE
fix documentation build warnings

### PR DESCRIPTION
**Pull request by Onvember**
This PR fixes some warnings while building the docs.

fixes part of #903

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -370,7 +370,7 @@ Fixed
 
 - Properly handle nested input object in GraphQL:
 
-  .. code-block:: graphql
+  .. code-block:: text
```
Comment (rosik) [inline CHANGELOG.rst:373]: That's not fair

File: rst/cartridge_dev.rst [hunk 2]
```diff
@@ -206,7 +206,7 @@ To implement a custom cluster role, do the following:
    .. code-block:: lua
 
       -- Implement a custom role in app/roles/custom-role.lua
-      #!/usr/bin/env tarantool
+      -- #!/usr/bin/env tarantool
```
Comment (rosik) [inline rst/cartridge_dev.rst:209]: Let's remove this line entirely, see #794 

```suggestion
```

**Review by rosik — COMMENTED** [review]
Please, rebase it on top of the fresh master branch.

File: CHANGELOG.rst (hunk 1 above)
Comment (NickVolynkin) [reply CHANGELOG.rst:419]: There's no `graphql` lexer provided with Pygments. However, we can try to use https://pypi.org/project/pygments-graphql/.

**Review by rosik — APPROVED** [review]

**Pull request merged**